cvmfs/cvmfs_server

```diff
@@ -899,7 +899,7 @@ close_transaction() {
     cvmfs_suid_helper rdonly_umount $name
   fi
   cvmfs_suid_helper clear_scratch $name
-  rm -rf "${tmp_dir}/*"
+  rm -rf ${tmp_dir}/*
```

Comment:
That should be prepended with a test if `${tmp_dir}` is defined.  Hm, I wonder why it does not work with the quotes?  If for some reason we ever have a space in `$tmp_dir`, this command will surprise us, too.